scripts/gen-aliasarr.template.sh

```
@@ -1,4 +1,4 @@
-# shellcheck disable=SC1090,SC2154
+# shellcheck disable=SC1090,SC2119,SC2120,SC2154,SC2155
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)